build/IPDK_Container/README

```diff
@@ -86,8 +114,11 @@ Commands to setup the example scenario:
        (your mounted directory)
 
 2) From within the container create 2 vhost ports using GNMI commands
-    a. (If new bash) Run - 
-       source /root/P4-OVS/p4ovs_env_setup.sh /root/p4-sde/install 
+    a. (If new bash) Run -
```

Comment:
We need to further optimize this procedure later to make it common instead of following different paths of the scripts.